### DISCUSSION TITLE
Hyperparameter change and model improvement!

### DISCUSSION THREAD

**Discussion by ehzawad** [coqui-ai/TTS#2074] [General]
I have **18 hours** of audio data, and I am currently using **VITS recipe**, Is it possible to know beforehand that there is a PARTICULAR recipe would work better for my case. Also, I was wondering on a particular value of **use_phonemes**, should I keep it True or change it to False. 

config = VitsConfig(
    model_args=vitsArgs,
    audio=audio_config,
    run_name="vits_vctk",
    batch_size=32,
    eval_batch_size=16,
    batch_group_size=5,
    num_loader_workers=4,
    num_eval_loader_workers=4,
    run_eval=True,
    test_delay_epochs=-1,
    epochs=1000,
    text_cleaner="english_cleaners",
    use_phonemes=True,
    phoneme_language="en",
    phoneme_cache_path=os.path.join(output_path, "phoneme_cache"),
    compute_input_seq_cache=True,
    print_step=25,
    print_eval=False,
    mixed_precision=True,
    max_text_len=325,  # change this if you have a larger VRAM than 16GB
    output_path=output_path,
    datasets=[dataset_config],
    cudnn_benchmark=False,
)

So far, my model's training progress is something like this: (left: Average loss duration; right Average loss feature)
![loss](https://user-images.githubusercontent.com/12381692/195319194-459424f9-bbc0-48da-9327-7166f18a99a3.png)

Any pointer on this would be appreciated!

**Comment by Ca-ressemble-a-du-fake** [comment]
I keep it to true and specify `phonemizer="espeak"` in my recipe (although this is the default value). I don't pay much attention to the graphs while training with VITS, I prefer to listen to the test audios that are available in tensorboard.

**Reply by ehzawad** [reply]
http://sndup.net/bv5s

After training my model with VITS, I have got this (listenable audio is shared in the link)
Original Text: মিনিবাস থেকে নেমে জায়গাটা মোটেই পছন্দ হলো না ভাস্করের। (close English transliteration of that Bengali line) 

Minibāsa thēkē nēmē jāẏagāṭā mōṭē'i pachanda halō nā bhāskarēra.

Any suggestion on this?
